libraries/AP_Mission/AP_Mission.cpp

```diff
@@ -914,12 +914,16 @@ MAV_MISSION_RESULT AP_Mission::mavlink_int_to_mission_cmd(const mavlink_mission_
         break;
 
     case MAV_CMD_NAV_LAND:                              // MAV ID: 21
+#if APM_BUILD_TYPE(APM_BUILD_ArduPlane)
         cmd.p1 = packet.param1;                         // abort target altitude(m)  (plane only)
         cmd.content.location.loiter_ccw = is_negative(packet.param4); // yaw direction, (plane deepstall only)
+#endif
```

Comment:
I think it would be better to fix is_negative() to return false for NaN

Reply:
Addressed.

Reply:
Should `loiter_ccw` default to something if the parameter is being transmitted as nan?